wake_t/physics_models/plasma_wakefields/qs_rz_baxevanis_ion/plasma_particle_container.py

```diff
@@ -0,0 +1,192 @@
+"""Contains the definition of the `PlasmaParticleContainerPy` class."""
+
+import numba
+
+
+class PlasmaParticleContainerPy:
+    # Particle properties
+    r: numba.float64[::1]
```

Comment:
Could you add a short description of properties? If not all, at least those not obvious from the name
```suggestion
    r: numba.float64[::1] # radial position
```